__tests__/vender.spec.js

```diff
@@ -0,0 +1,99 @@
+const vendingMachine = require('./../src/vending-machine');
+
+describe('vendingMachine', () => {
+    let test;
+
+
+    beforeEach(() => {
+        test = {};
+
+        test.processData = {
+
+            coins: [
+                {
+                    type: 'dime',
+                    value: 0.10,
+                    amount: 100
+                },
+                {
+                    type: 'quarter',
+                    value: 0.25,
+                    amount: 50
+                },
+                {
+                    type: 'loonie',
+                    value: 1.00,
+                    amount: 1
+                },
+                {
+                    type: 'toonie',
+                    value: 2.00,
+                    amount: 10
+                },
+            ],
+
+            invintory: [
```

Comment:
I've noticed that spelling can sometimes be an issue for you. This can result in bugs. I'd recommend using a built-in spell checker for VSCode. I use Code Spellchecker